server/server.go

```diff
@@ -592,6 +595,78 @@ func (s *server) CreateBlockchains(
 	return &rpcpb.CreateBlockchainsResponse{ClusterInfo: s.clusterInfo}, nil
 }
 
+func (s *server) CreateSpecificBlockchains(
+	ctx context.Context,
+	req *rpcpb.CreateSpecificBlockchainsRequest,
+) (*rpcpb.CreateSpecificBlockchainsResponse, error) {
+	// if timeout is too small or not set, default to 5-min
+	if deadline, ok := ctx.Deadline(); !ok || time.Until(deadline) < defaultStartTimeout {
+		var cancel context.CancelFunc
+		ctx, cancel = context.WithTimeout(context.Background(), defaultStartTimeout)
+		_ = cancel // don't call since "start" is async, "curl" may not specify timeout
+		s.log.Info("received start request with default timeout", zap.String("timeout", defaultStartTimeout.String()))
+	} else {
+		s.log.Info("received start request with existing timeout", zap.String("timeout", deadline.String()))
+	}
+
+	s.log.Debug("CreateSpecificBlockchains")
+
+	if s.getNetwork() == nil {
+		return nil, ErrNotBootstrapped
+	}
+
+	if len(req.GetBlockchainSpecs()) == 0 {
+		return nil, ErrNoBlockchainSpec
+	}
+
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	chainSpecs := []network.BlockchainSpec{}
+	for _, spec := range req.GetBlockchainSpecs() {
+		chainSpec, err := getNetworkBlockchainSpec(s.log, spec, false, s.network.pluginDir)
+		if err != nil {
+			return nil, err
+		}
+		if len(chainSpec.Participants) == 0 {
+			return nil, ErrNoParticipants
+		}
+		if chainSpec.SubnetID != nil {
+			return nil, ErrSubnetNotNil
+		}
+		chainSpecs = append(chainSpecs, chainSpec)
+	}
+	if len(chainSpecs) == 0 {
+		return nil, ErrNoBlockchainSpec
+	}
+
+	chains, err := s.network.createSpecificBlockchains(ctx, req.ExecPath, chainSpecs, s.cfg.RedirectNodesOutput)
```

Comment:
Always have preferred all of this to be blocking, but grpc server functions were async since start. I think that was to try to easy certain client apps. But usually our client apps just prefer to block. Anyway, we have the WaitForHealthy now, that also returns the async error. Maybe to be uniform, this new one should also be non blocking and force the user to also call WaitForHealthy?